src/shared/GitHub/GitHubConstants.cs

```diff
@@ -32,6 +32,11 @@ public static class GitHubConstants
         // TODO: update this with a real version number once the GCM OAuth application has been deployed to GHE
         public static readonly Version MinimumEnterpriseOAuthVersion = new Version("99.99.99");
```

Comment:
Once we know which version of GHES has the OAuth app deployed, we can update this constant too.

Reply:
Will this ever cause confusion since "Enterprise" is a part of the name of three different GitHub products? `MinimumOnPremOAuthVersion` maybe? (This may be pointless bikeshedding on my part - the question is genuine but not particularly important.)

Reply:
Feel free to submit the rename PR to that OnPrem suggestion ;)

Reply:
Ha 😁

Reply:
Whoops, can't resolve, since you had left this as a note to yourself.